app/internal/daily_quotes.py

```diff
@@ -1,11 +1,16 @@
 from datetime import date
-from typing import Dict, Optional
+from os.path import relpath
+from typing import Dict, List, Optional
 
 from sqlalchemy.orm import Session
 from sqlalchemy.sql.expression import func
 
-from app.database.models import Quote
+from app.database.models import Quote, UserQuotes
+from app.dependencies import MEDIA_PATH
 
+EMPTY_HEART_PATH = relpath(f"{MEDIA_PATH}\\empty_heart.png", "app")
+FULL_HEART_PATH = relpath(f"{MEDIA_PATH}\\full_heart.png", "app")
+FAVORITES_PATH = f"../../{FULL_HEART_PATH}"
```

Comment:
Same here. Also, prefer having it as an absolute path unless you pass it to the front

Reply:
I do pass it to the front.